lib/solidus_kustomer/client.rb

```diff
@@ -13,6 +13,44 @@ def initialize(api_key:, url: 'https://api.kustomerapp.com/v1/')
       @url = url
     end
 
+    # Creates a new customer on Kustomer
+    #
+    # @param user [Spree::User] the user to create on Kustomer
+    #
+    # @return [Hash] containing the newly created data on Kustomer
+    def create_customer(user)
+      response = HTTParty.post(
+        "#{@url}customers",
+        body: SolidusKustomer::Serializer::User.serialize(user).to_json,
+        headers: headers
+      )
+
+      raise(SolidusKustomer::CustomerCreateError) unless response.success?
+
+      user.update!(kustomer_id: response.parsed_response['data']['id'])
+      response.parsed_response['data']
+    end
+
+    # Updates a customer on Kustomer
+    #
+    # @param user [Spree::User] the user to update on Kustomer
+    #
+    # @return [Hash] containing the updated Kustomer data or nil
+    # if it attempts to update a customer without a kustomer_id
+    def update_customer(user)
```

Comment:
I would call this `sync_customer`. Ruby/Rails conventions would dictate that a method called `update_customer` also accepts a hash of properties to set on the customer, which is not the case here, because you're just syncing the properties on the `Spree::User` record with the respective customer in Kustomer.

Reply:
That makes a lot of sense. Will change it accordingly, thanks.